libraries/typings-generator/src/TypingsGenerator.ts

```diff
@@ -152,6 +185,21 @@ export class TypingsGenerator {
     }
   }
 
+  private _clearDependencies(target: string): void {
+    const dependencies: IterableIterator<string> | undefined = this._targetMap.get(target)?.keys();
+    if (dependencies) {
+      for (const dependency of dependencies) {
+        // eslint-disable-next-line no-unused-expressions
+        this._dependencyMap.get(dependency)?.delete(target);
```

Comment:
Given the code above that adds the dependencies, would this ever be a case where the dependency would be `undefined` anyway? If not, it would be clearer to do `this._dependencyMap.get(dependency)!.delete(target)`

Reply:
You are correct, it should be impossible for it to be undefined. Updated.